Jellyfin.Plugin.OpenSubtitles/API/OpenSubtitlesController.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Net.Mime;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using OpenSubtitlesHandler;
+using OpenSubtitlesHandler.Models;
+
+namespace Jellyfin.Plugin.OpenSubtitles.API
+{
+    /// <summary>
+    /// The open subtitles plugin controller.
+    /// </summary>
+    [ApiController]
+    [Produces(MediaTypeNames.Application.Json)]
+    public class OpenSubtitlesController : ControllerBase
+    {
+        /// <summary>
+        /// Validates login info.
+        /// </summary>
+        /// <remarks>
+        /// Accepts plugin configuration as JSON body.
+        /// </remarks>
+        /// <response code="200">Login info valid.</response>
+        /// <response code="400">Login info is missing data.</response>
+        /// <response code="401">Login info not valid.</response>
+        /// <param name="body">The request body.</param>
+        /// <param name="token">Cancellation token.</param>
+        /// <returns>
+        /// An <see cref="NoContentResult"/> if the login info is valid, a <see cref="BadRequestResult"/> if the request body missing is data
+        /// or <see cref="UnauthorizedResult"/> if the login info is not valid.
+        /// </returns>
+        [HttpPost("Jellyfin.Plugin.OpenSubtitles/ValidateLoginInfo")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        public async Task<ActionResult> ValidateLoginInfo([FromBody] LoginInfoInput body, CancellationToken token)
```

Comment:
```suggestion
        public async Task<ActionResult> ValidateLoginInfo([FromBody] LoginInfoInput body)
```

I don't think having a CancellationToken as a controller param is useful. Use `CancellationToken.None` if needed

Reply:
Removed in 8b64d9997bdc5a4b0eca43019f26daf0b6045052